continuous_integration/gpuci/build.sh

```diff
@@ -55,4 +66,4 @@ conda config --show-sources
 conda list --show-channel-urls
 
 gpuci_logger "Python py.test for dask-sql"
-py.test $WORKSPACE -n 4 -v -m gpu --rungpu --cov-config="$WORKSPACE/.coveragerc" --cov=dask_sql --cov-report=xml:"$WORKSPACE/dask-sql-coverage.xml" --cov-report term
+py.test $WORKSPACE -n 4 -v -m gpu --runqueries --rungpu --junitxml="$WORKSPACE/junit-dask-sql.xml" --cov-config="$WORKSPACE/.coveragerc" --cov=dask_sql --cov-report=xml:"$WORKSPACE/dask-sql-coverage.xml" --cov-report term
```

Comment:
Is there a way to make the query tests another test in our suite that's optional rather than bundled with the regular tests?
If not that's okay and we can probably skip/ignore failing queries for now.

Reply:
Yeah, two options I could think of are:

1. moving the test file to a different directory from `tests/` like `queries/`, so that calling `pytest tests` would skip over it in the search entirely
2. not include the file in the repo, instead pulling in from somewhere else during the gpuCI run 

My personal preference would be 1 (though it's not perfect, running just `pytest` would still catch the queries) since we don't have a clear place to publish the testing files outside of this repo